src/nvim/undo.c

```diff
@@ -1487,7 +1498,83 @@ void u_read_undo(char_u *name, char_u *hash, char_u *orig_name)
   return;
 }
 
+/// Writes a sequence of bytes to the undo file.
+///
+/// @returns false in case of an error.
+static bool undo_write(bufinfo_T *bi, uint8_t *ptr, size_t len)
+{
+  return fwrite(ptr, len, 1, bi->bi_fp) == 1;
+}
+
+/// Writes a number, MSB first, in "len" bytes.
```

Comment:
Maybe it's just me, but high-order bit seems more common, or even most significant bit.